lib/policy_machine/version.rb

```diff
@@ -1,3 +1,3 @@
 class PolicyMachine
-  VERSION = "3.1.0"
+  VERSION = "3.1.1"
```

Comment:
opinions on version number?  couldn't decide between `3.1.1` and `3.2.0`.

Reply:
The changelog seems to be inconsistent about when an added method merits a minor bump vs point bump. In a vacuum, I think this would receive a minor version change for updating the outward facing API and reserve points for small tweaks?

That being said, it seems relatively arbitrary.

Reply:
Thanks, yea the "minor version for change in outward facing API" makes sense to me.  Will go with `3.2.0`.